openlibrary/plugins/books/readlinks.py

```diff
@@ -33,8 +33,8 @@ def ol_query(name, value):
 
 def get_solr_select_url():
     c = config.get("plugin_worksearch")
-    host = c and c.get('solr')
-    return host and ("http://" + host + "/solr/select")
+    base_url = c and c.get('solr_base_url')
```

Comment:
Should we raise or at least log `if not base_url:` ?  This seems similar to our most frequent Sentry issue.

Reply:
Not sure if this is the best place in the code for that; I think having a better interface for accessing config that manages those sorts of things would be better.

Opting to leave as-is since it was a pre-existing issues :+1: